.github/workflows/pull_request.yml

```diff
@@ -38,6 +38,7 @@ jobs:
     - uses: actions/checkout@master
     - name: Run Snyk to check for vulnerabilities
       uses: snyk/actions/node@master
+      continue-on-error: true
```

Comment:
I prefer to the test to fails and see the report then continue on error, even if it exists in other repos.

Reply:
this is a known [issue](https://github.com/actions/runner/issues/2347) which sadly github-workflow does not support. 
Notice that the report is shown but the step is marked with a green tick.